src/sources/exec/mod.rs

```diff
@@ -688,6 +830,50 @@ mod tests {
         assert!(log.get(log_schema().timestamp_key()).is_some());
     }
 
+    #[test]
+    fn test_streaming_create_event_vector_namespace() {
+        let config = standard_streaming_test_config();
+        let hostname = Some("Some.Machine".to_string());
+        let data_stream = Some(STDOUT.to_string());
+        let pid = Some(8888_u32);
+
+        let mut event = LogEvent::from("hello world").into();
```

Comment:
I _think_ this shouldn't be used in tests anymore - this is something I'm actively looking at in the kubernetes logs source though, so might not be correct.

Reply:
🤔 since the default `decoding` is `bytes` the vector namespace should have this as `. = "hello world"`, not `.message = "hello world"`.

You can use: 

```rust
impl From<Value> for LogEvent {
    fn from(value: Value) -> Self {
        Self::from_parts(value, EventMetadata::default())
    }
}
```

or 

```rust
LogEvent::from_parts(value!("hello world"), EventMetadata::default())
```

Reply:
This applies to the other test as well

Reply:
Is this just another, more recent style, of arriving at the same thing? The rest of the tests use `LogEvent::from`

Reply:
By changing the LogEvent to have an inner Bytes Value, the types within the test change some: we go from having the message within the "message" key to having it as the value of the log. Would that be fine?

Reply:
Yeah, that's why the change is needed: https://cs.github.com/vectordotdev/vector/blob/787347b433fa0fc40fcd5b1a774acd290671c43b/lib/vector-core/src/event/log_event.rs#L420-L424

`LogEvent::from("hello world")`:
```coffee
. = {
  "message": "hello world"
}
```

`LogEvent::from(vrl::value!("hello world"))`:
```coffee
. = "hello world"
```

Part of the transition from Legacy => Vector namespaces is formerly the event was _always_ an object, with the new namespace that's not a requirement. For example, the [BytesDeserializer](https://cs.github.com/vectordotdev/vector/blob/787347b433fa0fc40fcd5b1a774acd290671c43b/lib/codecs/src/decoding/format/bytes.rs#L37) will parse the incoming bytes into a "message" field with the Legacy namespace, but directly into the event root for Vector.

Reply:
Great, thanks for catching that.